shell/browser/api/electron_api_web_frame_main.h

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2020 Samuel Maddock <sam@samuelmaddock.com>.
+// Use of this source code is governed by the MIT license that can be
+// found in the LICENSE file.
+
+#ifndef SHELL_BROWSER_API_ELECTRON_API_WEB_FRAME_MAIN_H_
+#define SHELL_BROWSER_API_ELECTRON_API_WEB_FRAME_MAIN_H_
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "gin/handle.h"
+#include "gin/wrappable.h"
+
+class GURL;
+
+namespace content {
+class RenderFrameHost;
+}
+
+namespace gin {
+class Arguments;
+}
+
+namespace gin_helper {
+class Dictionary;
+}
+
+namespace electron {
+
+namespace api {
+
+// Bindings for accessing frames from the main process.
+class WebFrame : public gin::Wrappable<WebFrame> {
```

Comment:
Consistency says this class should actually be called `WebFrameMain`

Reply:
7980039